nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const app = express();
+
+// import data here
+const meals = require('./data/meals.json');
+const reservations = require('./data/reservations.json');
+const reviews = require('./data/reviews');
+
+app.get('/', sendGreeting);
+app.get('/meals', listMeals);
+app.get('/reservations', listReservations);
+app.get('/cheap-meals', cheapMealsList);
+app.get('/large-meals', largeMealsList);
+app.get('/meal', listMeal);
+app.get('/reservation', listReservation);
+
+async function sendGreeting(request, response) {
+    response.send('asd');
+}
+
+async function listMeals(request, response) {
+    response.json(getMapAndFilter(meals));
+}
+
+function getMapAndFilter(param) {
```

Comment:
You could call this argument meals, you know it will always be meals.
Similarly, the name of the function tells me the implementation of the code (it uses map and filter) but not the business value.
I'd suggest a name such as "joinMealsWithReviews" since that's what you're really doing.